examples/nginx-app/with-pv.yaml

```diff
@@ -70,15 +70,20 @@ spec:
           - mountPath: "/var/log/nginx"
             name: nginx-logs
             readOnly: false
-      - image: velero/fsfreeze-pause:latest
+      - image: ubuntu:bionic
         name: fsfreeze
         securityContext:
           privileged: true
         volumeMounts:
           - mountPath: "/var/log/nginx"
             name: nginx-logs
             readOnly: false
+        command:
+          - "/bin/bash"
+          - "-c"
+          - "while true; do sleep 10000; done"
```

Comment:
would it be better to `tail -f /dev/null`?

Reply:
Looks like `sleep infinity` is another option too

Reply:
Yeah we could switch to one of those - we had the loop because we used to use alpine/busybox which didn't support `sleep infinity`. The internet has varying opinions on which is the most efficient - I'll take a look and decide. TBH I think any are probably fine in this context :)

Reply:
switched to `sleep infinity`.